PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -1,41 +1,15 @@
 <template>
 	<v-container fluid>
 
-
+    <v-row align="center" justify="start">
+      <v-col cols="12">
+      <div class="font-weight-regular">Use this QR code to easily check-in at the site where you receive your vaccine. This QR code contains an encrypted patient identifier so we can quickly and securely identify you and retrieve your information.</div></v-col></v-row> 
     <v-row justify="center">
-    <v-col class="d-flex" cols="10" sm="8" md="6" lg="6">
-  <v-text-field
-      v-model="qrText"
-      clearable
-      counter
-      filled
-      label="Type some text here"
-      @input="generateQrCode"
-    ></v-text-field>
-    </v-col></v-row>
-
-    <v-row justify="center">
-<div class="justify-center" v-if="qrSrc">
-  <img class="preview" :src="qrSrc"> </div>
-    </v-row>
-
-    <v-row>
-    <v-btn
-      color="accent"
-      :disabled="!qrSrc"
-      @click="reset"
-    >
-    Reset
-    </v-btn>
-    <v-spacer></v-spacer>
-    <v-btn
-    color="primary"
-      :disabled="!qrSrc"
-      @click="openInNewWindow"
-    >
-    Open 
-    </v-btn>
-    </v-row>
+      <div>
+				<vue-qrcode
+					v-bind:value="qrValue"
```

Comment:
I'm still getting the console error where qrValue "is not defined on the instance but referenced during render." I imagine you're using qrValue during startup somewhere since I immediately ran into this error upon selecting "Register Myself" on the first page so this probably has to do something with how you're defined/use qrValue.